src/test/resources/bundles/defaultProcess_V1/config.json

```diff
@@ -109,6 +110,7 @@
         "style"
       ],
       "acknowledgmentAllowed": "Always",
+      "showDetailCardHeader" : true,
```

Comment:
The default value is true so it is not needed, it "weighs down" unnecessarily the file.